Assets/MixedRealityToolkit/_Core/Devices/OpenVR/GenericOpenVRController.cs

```diff
@@ -66,6 +72,8 @@ public GenericOpenVRController(TrackingState trackingState, Handedness controlle
             // Oculus Touch Controller - Button.Four Press
             // Valve Knuckles Controller - Left Controller Outer Face Button
             new MixedRealityInteractionMapping(9, "Unity Button Id 3", AxisType.Digital, DeviceInputType.ButtonPress, MixedRealityInputAction.None, KeyCode.JoystickButton3),
+            new MixedRealityInteractionMapping(10, "WMR Touchpad Touch", AxisType.Digital, DeviceInputType.TouchpadTouch, MixedRealityInputAction.None, KeyCode.JoystickButton18),
```

Comment:
I see the WMR touchpad touch coming through `new MixedRealityInteractionMapping(6, "Trackpad-Thumbstick Touch", AxisType.Digital, DeviceInputType.TouchpadTouch, MixedRealityInputAction.None, KeyCode.JoystickButton16)` as expected. Do we need this?

Reply:
Huh, I'll double check. I might have missed this one.

Reply:
![image](https://user-images.githubusercontent.com/13334553/43804017-77e322a4-9a4f-11e8-8462-75fc85904c52.png)

Reply:
Oh I see what I did here, I got it confused with the axes haha.

Reply:
Well, it is touchpad press for WMR but the other devices have it categorized as touch.

Reply:
I think these mappings are only valid for UWP. I'm not getting anything anything through these new mappings on Standalone/OpenVR. Are you able to run this?

Reply:
I tested recently with good results, but I'm going down a different path for this. You'll see in an update soon.